packages/manager/src/features/CloudPulse/Alerts/Utils/utils.ts

```diff
@@ -204,3 +230,28 @@ export const convertAlertDefinitionValues = (
     trigger_conditions,
   };
 };
+
+/**
+ *
+ * @param criterias list of metric criterias to be processed
+ * @returns list of metric criterias in processed form
+ */
+export const processMetricCriteria = (
+  criterias: AlertDefinitionMetricCriteria[]
+): ProcessedCriteria[] => {
+  return criterias
+    .map((criteria) => {
```

Comment:
```suggestion
    .map(({aggregate_function, label, operator, threshold, unit}) => {
```
you can directly de-structure here and use it as well